src/apps/relay/dbdrivers/dbd_mysql.c

```diff
@@ -44,7 +44,7 @@ struct _Myconninfo {
   char *dbname;
   char *user;
   char *password;
-  unsigned int port;
+  uint16_t port;
```

Comment:
I recommend caution in changing the dbdrivers code. It's not guaranteed that a database protocol uses the same underlying concept of a port that the TCP and UDP protocols use.


You might want to defer any changes to the dbdrivers to another pull request in the future.